File: docs/components_page/components/form/inline.R
library(dashBootstrapComponents)

form <- dbcForm(
  dbcRow(
    list(
      dbcLabel("Email", width = "auto"),
      dbcCol(
        dbcInput(type = "email", placeholder = "Enter email"),
        className = "me-3"
      ),
      dbcLabel("Password", width = "auto"),
      dbcCol(
        dbcInput(type = "password", placeholder = "Enter password"),
        className = "me-3"
      ),
      dbcCol(dbcButton("Submit", color = "primary"), width = "auto")
    ),
    className = "g-2"
  )
)

File: docs/components_page/components/form/inline.jl
using DashBootstrapComponents

form = dbc_form(
    dbc_row(
        [
            dbc_label("Email", width = "auto"),
            dbc_col(
                dbc_input(type = "email", placeholder = "Enter email"),
                className = "me-3",
            ),
            dbc_label("Password", width = "auto"),
            dbc_col(
                dbc_input(type = "password", placeholder = "Enter password"),
                className = "me-3",
            ),
            dbc_col(dbc_button("Submit", color = "primary"), width = "auto"),
        ],
        className = "g-2",
    ),
)
